CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ cmake_minimum_required(VERSION 3.24)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
+option(BUILD_WASM "Build CXX-Qt for WASM" OFF)
```

Comment:
probably change to `BUILD_TESTS` and `ON` by default

Reply:
we'll probably still need BUILD_WASM to specify wasm-only stuff, right?

maybe have

```cmake
if(NOT BUILD_WASM)
    set(BUILD_TESTS OFF)
endif()
```

?

Reply:
I guess unless there is another way to tell we are under wasm/emscripten from CMake ?